frameworks/AutoGluon/exec.py

```diff
@@ -48,6 +48,17 @@ def run(dataset, config):
 
     is_classification = config.type == 'classification'
     training_params = {k: v for k, v in config.framework_params.items() if not k.startswith('_')}
+    if "high_quality" in training_params.get("presets", []):
```

Comment:
No, because `medium_quality` also doesn't refit_full.

`high_quality` and `good_quality` use refit_full

Reply:
There technically isn't a problem with using `optimize_for_deployment` with all presets, but if you are tracking the models trained, you would lose unused models from that, because they are immediately deleted at end of training.

Reply:
aka for non-interactive end-to-end black box usage of AutoGluon, it is always optimal to use `optimize_for_deployment`

Reply:
(Ignore previous 2 comments, I thought this was the `optimize_for_deployment` PR).